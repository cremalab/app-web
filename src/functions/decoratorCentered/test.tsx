import React from "react"
import { decoratorCentered } from "."

describe("decoratorCentered", () => {
  it("works", () => {
    // Arrange
    const story = () => <div>test</div>
    const context = { kind: "kind", story: "test" }

    // Act
    const received = decoratorCentered(story, context)

    // Assert
    expect(received).toMatchSnapshot()
  })
})
