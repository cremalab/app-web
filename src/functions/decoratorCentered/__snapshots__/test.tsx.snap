// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`decoratorCentered works 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "bottom": 0,
      "display": "flex",
      "justifyContent": "center",
      "left": 0,
      "padding": "1em",
      "position": "absolute",
      "right": 0,
      "top": 0,
    }
  }
>
  <div>
    test
  </div>
</div>
`;
